backend/capellacollab/core/database/__main__.py

```diff
@@ -40,6 +42,7 @@ def migrate_db():
 
         context = MigrationContext.configure(conn)
         current_rev = context.get_current_revision()
+        tools_exist = inspect(engine).has_table('tools')
```

Comment:
I think it would be more intuitive this way: 
`if not inspect(engine).has_table('tools'):`

Reply:
The check must be performed before the tables are created, but the insertion must be performed after, hence the separation.

Reply:
Why do we need the check at all? When the migration scripts were successful, they should always be there, right?